pyproject.toml

```diff
@@ -142,6 +143,13 @@ simulation = [
     "xyzpy>=1.2.1",
 ]
 
+benchmarking = [
+    "baybe[chem]",
+    "baybe[onnx]",
+    "baybe[simulation]",
+    "GitPython==3.1.43",
```

Comment:
Why this very explicit version fix?

Reply:
I saw you turned it into >= but kept the very specific version number. Since this is not related to any algorithms, I'd say let's loosen this as much as possible. Depending on what you need from the package, it might be possible to drop the version altogether! If not, then I'd say let's go with the smallest major version possible